jbpm-wb-forms/jbpm-wb-forms-client/src/main/java/org/jbpm/workbench/forms/client/display/KieServerFormsStartProcessDisplayer.java

```diff
@@ -0,0 +1,158 @@
+
+    @Inject
+    protected Event<NewCaseInstanceEvent> newCaseInstanceEvent;
+    
+    private Frame inlineFrame = GWT.create(Frame.class);
+    
+
+    @Override
+    public Class<KieServerFormRenderingSettings> getSupportedRenderingSettings() {
+        return KieServerFormRenderingSettings.class;
+    }
+
+    @Override
+    protected void initDisplayer() {
+        startProcessCallback(this);
+        startCaseCallback(this);
+    }
+    
+    @Override
+    public void init(FormDisplayerConfig<ProcessDefinitionKey, KieServerFormRenderingSettings> config,
+                     Command onClose,
+                     Command onRefreshCommand) {
+        initConfigs(config,
+                    onClose,
+                    onRefreshCommand);
+
+        container.clear();
+        formContainer.clear();
+        footerButtons.clear();
+
+        container.add(formContainer);
+        
+        initDisplayer();
+
+        formContainer.add(getFormWidget());
+    }
+
+    @Override
+    public IsWidget getFormWidget() {                
+        
+        inlineFrame.setWidth("100%");
+        inlineFrame.setHeight("100%");
+        inlineFrame.getElement().setPropertyBoolean("webkitallowfullscreen",
+                                                    true);
+        inlineFrame.getElement().setPropertyBoolean("mozallowfullscreen",
+                                                    true);
+        inlineFrame.getElement().setPropertyBoolean("allowfullscreen",
+                                                    true);
+        inlineFrame.getElement().getStyle().setBorderWidth(0,
+                                                           Style.Unit.PX);
+        inlineFrame.getElement().getStyle().setPosition(Position.ABSOLUTE);
+        inlineFrame.getElement().getStyle().setTop(0, Unit.PX);
+        inlineFrame.getElement().getStyle().setLeft(0, Unit.PX);
+        
+        inlineFrame.setUrl(renderingSettings.getUrl());
+        
+        FlowPanel div = new FlowPanel();
+        div.getElement().getStyle().setPosition(Position.RELATIVE);
+        div.getElement().getStyle().setOverflow(Overflow.HIDDEN);
+        div.getElement().getStyle().setPaddingTop(40, Unit.PCT);
+        div.add(inlineFrame);
+        return div;
+    }
+
+    @Override
+    public void startProcessFromDisplayer() {
+        
+    }
+    
+    public void notifyAboutStartProcess(String id) {
+        
+        Long processInstanceId = Long.parseLong(id);
+        newProcessInstanceEvent.fire(new NewProcessInstanceEvent(serverTemplateId,
+                                                                 deploymentId,
+                                                                 processInstanceId,
+                                                                 processDefId,
+                                                                 processName,
+                                                                 1));
```

Comment:
can you pass null in here for the status, I recently refactored similar code, and status should not be part of the event.